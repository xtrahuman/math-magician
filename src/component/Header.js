import React from 'react';

const Header = () => (
  <header className="header">
    <h1>Math Magician</h1>
  </header>
);

export default Header;
