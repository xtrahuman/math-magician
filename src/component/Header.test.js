import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

it('renders correctly', () => {
  const heading = renderer
    .create(
      <Header />,
    )
    .toJSON();
  expect(heading).toMatchSnapshot();
});
