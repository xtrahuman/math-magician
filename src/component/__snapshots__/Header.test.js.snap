// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="header"
>
  <h1>
    Math Magicians
  </h1>
</header>
`;
